rebuild/build.ninja

```diff
@@ -1,350 +0,0 @@
-# This is the build file for project "eos-payg"
```

Comment:
Please rebase the branch and squash the patches together. This file should not be added in 1 commit and removed in another.